pdata/plog/generated_scopelogs.go

```diff
@@ -28,34 +28,60 @@ import (
 // This is a reference type, if passed by value and callee modifies it the
 // caller will see the modification.
 //
-// Must use NewScopeLogs function to create new instances.
+// Must use NewExemplar function to create new instances.
 // Important: zero-initialized instance is not valid for use.
 type ScopeLogs struct {
+	commonScopeLogs
+}
+
+type MutableScopeLogs struct {
+	commonScopeLogs
+	preventConversion struct{}
+}
+
+type commonScopeLogs struct {
```

Comment:
commonScopeLogs/immutableScopeLogs

Reply:
Do you mean renaming it?

Reply:
yes..

Reply:
Not sure I follow why we want to rename this?

Reply:
Even if we don't have methods on `commonScopeLogs` in this particular struct, it's not the case for other structs. Maybe better to keep it consistent and not introduce any additional logic to the generator looking at the set of struct fields and see if the `common` struct can be removed?